src/proxy/admin_shift.go

```diff
@@ -0,0 +1,133 @@
+package proxy
+
+import (
+	"config"
+	"flag"
+	"fmt"
+	"runtime"
+
+	"github.com/radondb/shift/build"
+	"github.com/radondb/shift/shift"
+	"github.com/radondb/shift/xlog"
+)
+
+const (
+	cleanup                = false
+	checksum               = true
+	mysqlDump              = "mysqldump"
+	threads                = 16
+	behinds                = 2048
+	radonURL               = "http://127.0.0.1:8080"
```

Comment:
In radon config, the value is `peer-address`. The define is `{RADON-HOST:8080}`.  We cannot be directly defined as a constant here.

Reply:
Good job, fixed.

Reply:
@hustjieke 
Review Pls.